pages/profile.php

```diff
@@ -12,14 +12,193 @@
+            <div class="col-xs-6 col-sm-4 placeholder">
+                <div style="height: 150px">
+                    <h2><?php echo $_SESSION["user"]->getFirstName() . " " . $_SESSION["user"]->getLastName() ?></h2>
+                    <h3 id="personal-message"><?php echo $_SESSION["user"]->getPersonalMessage() ?></h3>
+                </div>
+                <span class="text-muted">
+                    <a class="hover-pointer" data-toggle="modal" data-target="#pm-modal">
+                        <button type="button" class="btn btn-default" id="change-pm-modal">Change Personal Message</button>
+                    </a>
+                </span>
+            </div>
+        </div>
+
+        <div class="jumbotron dashboard-title">
+            <h3>
+                My Profile
+            </h3>
+        </div>
+
+        <h2 class="sub-header">Your Information</h2>
+
+        <div class="table-responsive">
+            <table class="table table-striped">
+                <thead>
+                <tr>
+                    <th>Username</th>
+                    <th>Password</th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr>
+                    <td><?php echo $_SESSION["user"]->getUsername(); ?></td>
+                    <td>
+                        <button type="button" class="btn btn-default" data-toggle="modal" data-target="#password-modal">
+                            Change Password
+                                    </button>
+                                </td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+            <div class="table-responsive">
+                <table class="table table-striped">
+                    <thead>
+                    <tr>
+                        <th>First Name</th>
+                        <th>Last Name</th>
+                        <th>Email Address</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <tr>
+                        <td><?php echo $_SESSION["user"]->getFirstName(); ?></td>
+                        <td><?php echo $_SESSION["user"]->getLastName(); ?></td>
+                        <td><?php echo $_SESSION["user"]->getEmail(); ?></td>
+                    </tr>
+                    </tbody>
+                </table>
+            </div>
+
+            <!-- Password Modal -->
+            <div id="password-modal" class="modal fade" role="dialog">
+                <div class="modal-dialog">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <h4 class="modal-title">Change your password</h4>
+                        </div>
+                        <div class="modal-body">
+                            <form id="changepassword">
+                                <label for="oldpassword" class="sr-only">Old Password</label>
```

Comment:
This is a mistake against some HTML design. Actually, a field should always have a label (for usability). The placeholder should not say what information goes there, it should be an example.

(e.g, instead of saying "email", it would say "email@hotmail.an"). But I'm okay with violating this "guideline", just thought you might want to know. :-)